Composer/packages/client/src/router.tsx

```diff
@@ -42,6 +43,11 @@ const BotCreationFlowRouter = React.lazy(() => import('./components/CreationFlow
 const BotCreationFlowRouterV2 = React.lazy(() => import('./components/CreationFlow/v2/CreationFlow'));
 const FormDialogPage = React.lazy(() => import('./pages/form-dialog/FormDialogPage'));
 
+const modalControlGroup = css`
+  border: 1px solid rgb(237, 235, 233);
+  padding: 0.5rem 1rem 1rem 1rem;
```

Comment:
Do we use rems anywhere in code? Can we stick to pixels?

Reply:
Changed to pixels!